mne/io/base.py

```diff
@@ -800,7 +800,7 @@ def __setitem__(self, item, value):
     @verbose
     def get_data(self, picks=None, start=0, stop=None,
                  reject_by_annotation=None, return_times=False, units=None,
-                 verbose=None):
+                 tmin=None, tmax=None, verbose=None):
```

Comment:
Easy way to prevent people from doing something silly
```suggestion
                 *, tmin=None, tmax=None, verbose=None):
```